max_angular_rate: 10
max_linear_rate: 10

change_mode_button: back_button

# parameters below are controlled by dynamic_reconfigure and these are only default values

mode: 0 # default steering mode
# normal=0, car=1, tank=2

gear: 1
gear_1_max_speed: 10
gear_2_max_speed: 50
gear_3_max_speed: 100
